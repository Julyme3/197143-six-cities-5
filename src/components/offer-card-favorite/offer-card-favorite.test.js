import React from "react";
import renderer from "react-test-renderer";
import OfferCardFavorite from "./offer-card-favorite";

jest.mock(`../offer-card/offer-card`, () => `OfferCard`);

const noop = () => {};

test(`OfferCardFavorite render`, () => {
  const tree = renderer
    .create(
        <OfferCardFavorite
          className="near-places__card"
          classNameInner="near-places__image-wrapper"
          onChangeActiveOffer={noop}
          width="100px"
          height="100px"
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
