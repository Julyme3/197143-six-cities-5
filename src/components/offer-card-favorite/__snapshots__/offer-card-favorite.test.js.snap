// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OfferCardFavorite render 1`] = `
<OfferCard
  className="near-places__card favorites__card"
  classNameInner="near-places__image-wrapper favorites__image-wrapper"
  height="100px"
  onChangeActiveOffer={[Function]}
  width="100px"
/>
`;
